src/backends/compose/compose_backend.py

```diff
@@ -82,10 +82,12 @@ def up(self, *_):
                 if process.stdout:
                     for line in process.stdout:
                         logger.info(line.decode().rstrip())
+            return True
         except Exception as e:
             logger.error(
                 f"An error occurred while executing `{' '.join(command)}` in {self.config_dir}: {e}"
             )
+            return False
```

Comment:
in "backend: remove ipv4 from compose backend" (https://github.com/bitcoin-dev-project/warnet/commit/50107b7f1dd88a1f0d393ed0e64b2ef8b19fddc7):

seems unrelated? (probably fine to keep it, just making sure im not missing something :sweat_smile: )

Reply:
It is, drive-by fixes not helping anything much